examples/step-18/doc/intro.dox

```diff
@@ -328,35 +329,84 @@ Both stress update and rotation are implemented in the function
 
 <h3>Parallel graphical output</h3>
 
-In the step-17 example program, the main bottleneck for %parallel computations
+In step-17, the main bottleneck for %parallel computations as far as run time
+is concerned
 was that only the first processor generated output for the entire domain.
 Since generating graphical output is expensive, this did not scale well when
-large numbers of processors were involved. However, no viable ways around this
-problem were implemented in the library at the time, and the problem was
-deferred to a later version.
+larger numbers of processors were involved. We will address this here.
 
-This functionality has been implemented in the meantime, and this is the time
-to explain its use. Basically, what we need to do is let every process
+Basically, what we need to do is let every process
 generate graphical output for that subset of cells that it owns, write them
 into separate files and have a way to display all files for a certain timestep
-at the same time. This way the code produces one .vtu file per processor per
+at the same time. This way the code produces one <code>.vtu</code> file per process per
 time step. The two common VTK file viewers ParaView and VisIt both support
-opening more than one .vtu file at once. To simplify the process of picking
+opening more than one <code>.vtu</code> file at once. To simplify the process of picking
 the correct files and allow moving around in time, both support record files
 that reference all files for a given timestep. Sadly, the record files have a
 different format between VisIt and Paraview, so we write out both formats.
 
-The code will generate the files solution-TTTT.NNN.vtu, where TTTT is the
-timestep number (starting from 1) and NNN is the processor id (starting from
+The code will generate the files <code>solution-TTTT.NNN.vtu</code>, 
+where <code>TTTT</code> is the
+timestep number (starting from 1) and <code>NNN</code> is the process rank
+(starting from
 0). These files contain the locally owned cells for the timestep and
-processor. The files solution-TTTT.visit is the visit record for timestep
-TTTT, while solution-TTTT.pvtu is the same for ParaView. Finally, the file
-solution.pvd is a special record only supported by ParaView that references
+processor. The files <code>solution-TTTT.visit</code> is the visit record for timestep
+<code>TTTT</code>, while <code>solution-TTTT.pvtu</code> is the same for ParaView. Finally, the file
```

Comment:
visit can also open `.pvtu`. Actually i did not even know there is a special output for `Visit`.